2023-04-14 14:55:04,267 INFO    MainThread:33874 [wandb_setup.py:_flush():68] Configure stats pid to 33874
2023-04-14 14:55:04,267 INFO    MainThread:33874 [wandb_setup.py:_flush():68] Loading settings from /Users/emilywesel/.config/wandb/settings
2023-04-14 14:55:04,267 INFO    MainThread:33874 [wandb_setup.py:_flush():68] Loading settings from /Users/emilywesel/Documents/GitHub/NCANDA_Contrastive/wandb/settings
2023-04-14 14:55:04,267 INFO    MainThread:33874 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-04-14 14:55:04,267 INFO    MainThread:33874 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program': 'main.py'}
2023-04-14 14:55:04,267 INFO    MainThread:33874 [wandb_init.py:_log_setup():476] Logging user logs to /Users/emilywesel/Documents/GitHub/NCANDA_Contrastive/wandb/run-20230414_145504-1n0mgz45/logs/debug.log
2023-04-14 14:55:04,268 INFO    MainThread:33874 [wandb_init.py:_log_setup():477] Logging internal logs to /Users/emilywesel/Documents/GitHub/NCANDA_Contrastive/wandb/run-20230414_145504-1n0mgz45/logs/debug-internal.log
2023-04-14 14:55:04,268 INFO    MainThread:33874 [wandb_init.py:init():516] calling init triggers
2023-04-14 14:55:04,268 INFO    MainThread:33874 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2023-04-14 14:55:04,268 INFO    MainThread:33874 [wandb_init.py:init():569] starting backend
2023-04-14 14:55:04,268 INFO    MainThread:33874 [wandb_init.py:init():573] setting up manager
2023-04-14 14:55:04,276 INFO    MainThread:33874 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-04-14 14:55:04,282 INFO    MainThread:33874 [wandb_init.py:init():580] backend started and connected
2023-04-14 14:55:04,293 INFO    MainThread:33874 [wandb_init.py:init():658] updated telemetry
2023-04-14 14:55:04,322 INFO    MainThread:33874 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2023-04-14 14:55:11,396 WARNING MsgRouterThr:33874 [router.py:message_loop():77] message_loop has been closed
2023-04-14 14:55:13,000 ERROR   Thread-1  :33874 [internal_api.py:execute():212] 502 response executing GraphQL.
2023-04-14 14:55:13,000 ERROR   Thread-1  :33874 [internal_api.py:execute():213] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2023-04-14 14:55:19,402 ERROR   MainThread:33874 [wandb_init.py:init():1105] error
Traceback (most recent call last):
  File "/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1078, in init
    run = wi.init()
  File "/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 697, in init
    result = handle.wait(
  File "/Users/emilywesel/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 259, in wait
    raise MailboxError("transport failed")
wandb.errors.MailboxError: transport failed
